src/Functions.tsx

```diff
@@ -74,13 +74,18 @@ const collectNearbyRowsByZoom = (
     const siblingRows = parentCombinedRow.rows;
     const newCoordinates = computeAverageCoordinates(siblingRows, givenRow);
     const newRows = updateExistingRows(siblingRows, givenRow);
-    combinedRows[parentIndex] = createCombinedRow(newCoordinates, newRows);
+    combinedRows[parentIndex] = createCombinedRow(
```

Comment:
Can we rename this file? (I should've caught this earlier) Since you're doing functional react, every file you write is a collection of functions, so this file name is very vague. If there is a clear, specific name, use that. If this collection of functions doesn't have a coherent theme, split this file up into smaller files of related functions.

Reply:
Would Utilities or utils make for a good rename? I'm thinking all these functions are ones that don't take in props (unlike MapChart, OrphanTableRows, PlaceIcon, RowMarker, SideBar, ZoomControl), and they're used to help create the components.

Reply:
Utilities suffers from a lot of the same issue - every function has utility, or else you wouldn't write it. If all the functions help render stuff, maybe `renderers.tsx`?